_episodes/04-package-structure.md

```diff
@@ -386,12 +405,51 @@ from .pdb import open_pdb
 from .xyz import open_xyz, write_xyz
 ```
 {: .language-python}
-The `__init__.py` file contains python code that is called when a module is imported. These lines are relative import statments to the functions within the `io` package. Think of them as pointers to the functions, i.e. when we look at the `io` package, it directs us to the location of the underlying functions, so we do not need to look within each submodule. This allows us to use the following import statement to access the functions:
+The `__init__.py` file contains python code that is called when a module is imported. These lines are relative import statements to the functions within the `io` package. Think of them as pointers to the functions, i.e. when we look at the `io` package, it directs us to the location of the underlying functions, so we do not need to look within each submodule. This allows us to use the following import statement to access the functions:
 ```
 from molecool.io import open_pdb, open_xyz, write_xyz
 ```
 {: .language-python}
 
+Although this is easier, it is still different from how we call all the other functions in our code. If we wanted the io functions to mimic the imports from the rest of the modules, we would need to modify the init from the main folder as well
+
+~~~
+from .functions import *
+from .measure import calculate_distance, calculate_angle
+from .molecule import build_bond_list
+from .visualize import draw_molecule, bond_histogram
+import molecool.io
+~~~
+{: .language-python}
+
+
+This would allow us now to call the functions with 
+
+~~~
+>>> molecool.io.open_pdb()
+~~~
+{: .language-python}
+
+And we could even make these functions more accessible by removing the need for the  `io`
+
+~~~
+from .functions import *
+from .measure import calculate_distance, calculate_angle
+from .molecule import build_bond_list
+from .visualize import draw_molecule, bond_histogram
+from .io.pdb import *
```

Comment:
I would prefer to not do `*` here.

Reply:
Also, if our `__init__` file in `io` imports the functions (`open_pdb`, `open_xyz`), I think we will not need to specify the module. We can just do `from .io import *`